src/highdicom/sr/templates.py

```diff
@@ -3516,23 +3681,24 @@ def __init__(
             relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT
         )
         self.append(modality_item)
-        frame_of_reference_uid_item = UIDRefContentItem(
-            name=CodedConcept(
-                value='112227',
-                meaning='Frame of Reference UID',
-                scheme_designator='DCM'
-            ),
-            value=frame_of_reference_uid,
-            relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT
-        )
-        self.append(frame_of_reference_uid_item)
+        if 'FrameOfReferenceUID' in image:
+            frame_of_reference_uid_item = UIDRefContentItem(
+                name=CodedConcept(
+                    value='112227',
+                    meaning='Frame of Reference UID',
+                    scheme_designator='DCM'
+                ),
+                value=image.FrameOfReferenceUID,
+                relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT
+            )
+            self.append(frame_of_reference_uid_item)
         pixel_data_rows_item = NumContentItem(
             name=CodedConcept(
                 value='110910',
                 meaning='Pixel Data Rows',
                 scheme_designator='DCM'
             ),
-            value=pixel_data_rows,
+            value=image.Rows,
             relationship_type=RelationshipTypeValues.HAS_ACQ_CONTEXT,
             unit=CodedConcept(
                 value='{pixels}',
```

Comment:
This seems wrong...

Reply:
Not sure what's incorrect here - the value should be the [number of rows in the pixel data of the image.](https://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_D.html#DCM_110910). The relationship with the parent should be 'Has Acquisition Context'.  What am I missing?

Reply:
It's the code used to describe the unit that I'm unsure about, i.e. `'{pixels}'`. Initially I thought you were trying to do a python f-string but looking into UCUM a bit more, I see that the curly braces denote some sort of annotation.

Technically speaking of course, this has no unit since it's simply a count. But if we think this is the right way to do it, we can go ahead with this

Reply:
I think it is the right way because otherwise the DicomSRValidator.sh tool reports it as an error.